ge_releaser/constants.py

```diff
@@ -13,7 +13,7 @@
 
 
 class GxURL(str, enum.Enum):
-    AZURE_BUILDS = f"https://dev.azure.com/{GITHUB_REPO}/_build?definitionId=1"
+    GITHUB_ACTIONS_BUILD = f"https://github.com/{GITHUB_REPO}/actions/workflows/ci.yml"
```

Comment:
```suggestion
    GITHUB_ACTIONS_BUILD = f"https://github.com/{GITHUB_REPO}/actions/workflows/ci.yml?query=event%3Apush"
```

Reply:
You're absolutely right. Thanks for the suggestion